src/app/users/UserHome.jsx

```diff
@@ -55,21 +55,14 @@ class UserHome extends React.Component {
       <ul>
         <li>
           <div>
-            Chef:
-            {' '}
-            {chef.chefUsername}
-, Address:
-            {' '}
-            {chef.chefAddress}
-, Description:
-            {' '}
-            {chef.chefDescription}
-, Hours:
-            {chef.chefHours}
+            {`Chef: ${chef.username}, Address: ${chef.streetAddress} ${chef.city}, ${
+              chef.stateName
+            }, ${chef.zip}, Description: ${chef.description}`}
+
             <Link
               to={{
                 pathname: '/user/chefdetails',
-                state: { username: this.state.username, currentChef: chef.chefUsername },
+                state: { username: this.state.username, currentChef: chef.username },
```

Comment:
Can you also pass the chefId and userId? I depend on these for queries, so I want to make sure they are available to pass to my components.

Reply:
In the details page I grab all the info.

Reply:
Awesome!